packages/dds/tree/src/test/sequence-change-family/testGeneratedChanges.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { strict as assert } from "assert";
+import * as fs from "fs";
+import { toDelta, Transposed as T } from "../../changeset";
+import { sequenceChangeRebaser } from "../../feature-libraries";
+import { verifyChangeRebaser } from "../../rebase";
+import { FieldKey } from "../../tree";
+import { brand } from "../../util";
+import { generateFuzzyCombinedChange } from "../rebase/fuzz";
+import { generateRandomChange, generateRandomUpPath } from "./randomSequenceGenerator";
+
+const fooKey = brand<FieldKey>("foo");
+const keySet = new Set([fooKey]);
+
+const pathGen = (seed: number) => generateRandomUpPath(keySet, seed, 3, 5);
+describe("testing the SequenceChangeRebaser", () => {
+    it("using generateRandomChange function", async () => {
+        const changes = new Set<T.LocalChangeset>();
+        for (let i = 0; i <= 15; i++) {
```

Comment:
Instead of generating sets of 15 changes and passing that to `verifyChangeRebaser`, I think we want to verify smaller sets (sets of 3 sounds like a good start) but we want to repeat the test many times (with a different seed passed to `generateRandomChange` each time).